.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish to PyPi
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+**"
```

Comment:
Why not `v.*` (or `*`)? This regex doesn't look like compatible with suffixes like `rc`. And are the two last symbols a part of some custom syntax?

Reply:
Also, the `.` between the numbers here isn't escaped, in regex a `.` character means any character, rather than a literal dot.

Reply:
This field is not a proper regex but a glob expression with character classes. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Then let's just use `v*`.

Reply:
```suggestion
      - "v*"
```

Reply:
Whoa, committing suggestions from myself also automatically resolves the conversation.